manifests/init.pp

```diff
@@ -126,5 +132,19 @@
       group  => $puppet_group,
       mode   => '0750',
     }
+    if $fact_watcher_service {
+      if $manage_fact_watcher_dependencies {
+        package { ['ruby-inotify', 'parallel']:
+          ensure   => 'installed',
+          provider => 'puppet_gem',
+          before   => Systemd::Unit_file['fact_watcher.service'],
```

Comment:
This before looks redundant if you already have `->` later on.

Reply:
fixed it